index.js

```diff
@@ -21,14 +21,17 @@ var toArray = Array.from || (function (slice) {
 
 // ===================================================================
 
-function eventToPromise (emitter, event, opts) {
-  var ignoreErrors = opts && opts.ignoreErrors
+function eventToPromise (emitter, event, _opts) {
+  var opts = _opts || {}
+  var ignoreErrors = opts.ignoreErrors
+  var errorEvent = opts.error || 'error'
```

Comment:
I think `opts.error` could be `null` to ignore error handling, what's your opinion?